webapp/app/__init__.py

```diff
@@ -77,7 +81,10 @@ def EUR(decimal):
 
 @app.context_processor
 def inject_template_globals():
-    return dict(plausible_domain=app.config['PLAUSIBLE_DOMAIN'])
+    return dict(
+        plausible_domain=app.config['PLAUSIBLE_DOMAIN'],
+        react_bundle_name=app.config['REACT_BUNDLE_NAME']
```

Comment:
This is probably also changing after the merge or is there a reason you do not use CONFIG.PLAUSIBLE_DOMAIN?

Reply:
Correct, yes.